src/react/account/details/properties/AccountKeys.jsx

```diff
@@ -0,0 +1,189 @@
+// @flow
+import { Banner, Button, Tooltip } from '@scality/core-ui';
+import React, { useEffect, useMemo } from 'react';
+import Table, * as T from '../../../ui-elements/Table';
+import { deleteAccessKey, listAccessKeys } from '../../../actions';
+import { useDispatch, useSelector } from 'react-redux';
+import { useFilters, useSortBy, useTable } from 'react-table';
+import type { Account } from '../../../../types/account';
+import type { AppState } from '../../../../types/state';
+import { Clipboard } from '../../../ui-elements/Clipboard';
+import { Warning } from '../../../ui-elements/Warning';
+import { formatDate } from '../../../utils';
+import { padding } from '@scality/core-ui/dist/style/theme';
+import styled from 'styled-components';
+
+type Props = {
+    account: Account,
+};
+
+const TableContainer = styled.div`
+    display: block;
+    width: fit-content;
+    margin-top: ${padding.large};
+`;
+
+const EllipsisCell = styled.a`
+  overflow: hidden;
+  overflow-wrap: break-word;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  word-break: break-all;
+`;
+
+const initialSortBy = [
+    {
+        id: 'created_at',
+        desc: true,
+    },
+];
+
+function AccountKeys({ account }: Props) {
+    const dispatch = useDispatch();
+    const accessKeysInfo = useSelector((state: AppState) => state.user.accessKeyList);
+
+    useEffect(() => {
+        dispatch(listAccessKeys(account.userName));
```

Comment:
If you want to retrieve account access keys, do not specify the username parameter. 
`If the UserName field is not specified, the user name is determined implicitly based on the AWS access key ID used to sign the request.`
You will have to retrieve the temporary access key of this specific account (using assumeRoleWithWebIdentity) to make the list access keys call.

Reply:
done